src/artm/messages.proto

```diff
@@ -163,6 +163,11 @@ message TopicSelectionThetaConfig {
   repeated float alpha_iter = 3;
 }
 
+// Represents a configuration of a iTopic-like modal Theta regularizer
+message iTopicThetaConfig {
```

Comment:
I was planning to implement the alternative mode for regularization later (so, it will have 'iTopic mode' and 'netPLSA mode'), and then rename it to something like ModalGraphTheta. 

I agree that name sounds mysterious, which is a bad thing.